integration-tests/benchmark/tests/keeper_test.go

```diff
@@ -0,0 +1,98 @@
+package benchmark
+
+//revive:disable:dot-imports
+import (
+	"math/big"
+
+	"github.com/smartcontractkit/chainlink-env/environment"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/chainlink"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/ethereum"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	eth_contracts "github.com/smartcontractkit/chainlink-testing-framework/contracts/ethereum"
+	"github.com/smartcontractkit/chainlink/integration-tests/actions"
+	"github.com/smartcontractkit/chainlink/integration-tests/contracts"
+	"github.com/smartcontractkit/chainlink/integration-tests/testsetups"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rs/zerolog/log"
+)
+
+var _ = Describe("Keeper benchmark suite @benchmark-keeper", func() {
+	var (
+		err                 error
+		testEnvironment     *environment.Environment
+		keeperBenchmarkTest *testsetups.KeeperBenchmarkTest
+		benchmarkNetwork    *blockchain.EVMNetwork
+	)
+
+	BeforeEach(func() {
+		By("Deploying the environment", func() {
+			benchmarkNetwork = blockchain.LoadNetworkFromEnvironment()
+			testEnvironment = environment.New(&environment.Config{InsideK8s: true})
+			err = testEnvironment.
+				AddHelm(ethereum.New(&ethereum.Props{
+					NetworkName: benchmarkNetwork.Name,
+					Simulated:   benchmarkNetwork.Simulated,
+					WsURLs:      benchmarkNetwork.URLs,
+				})).
+				AddHelm(chainlink.New(0, nil)).
+				AddHelm(chainlink.New(1, nil)).
+				AddHelm(chainlink.New(2, nil)).
+				AddHelm(chainlink.New(3, nil)).
+				AddHelm(chainlink.New(4, nil)).
+				AddHelm(chainlink.New(5, nil)).
+				Run()
+			Expect(err).ShouldNot(HaveOccurred())
+			log.Info().Str("Namespace", testEnvironment.Cfg.Namespace).Msg("Connected to Keepers Benchmark Environment")
+		})
+
+		By("Setup the Keeper test", func() {
+			chainClient, err := blockchain.NewEVMClient(benchmarkNetwork, testEnvironment)
+			Expect(err).ShouldNot(HaveOccurred(), "Connecting to blockchain nodes shouldn't fail")
+			keeperBenchmarkTest = testsetups.NewKeeperBenchmarkTest(
+				testsetups.KeeperBenchmarkTestInputs{
+					BlockchainClient:  chainClient,
+					NumberOfContracts: 500,
```

Comment:
nit. Maybe parameterise this as an argument to CLI command (with some defaults). (Similar with checkGas, performGas etc)?

Reply:
@kalverra I guess this is part of `Easier to run and configure` changes